lib/foreman_build_history/facet.rb

```diff
@@ -1,7 +1,2 @@
 Facets.register(BuildHistoryFacet) do
```

Comment:
Can you already add a tab to the host show page through facets? I tried with pagelets but the required mount points were missing as well.